pkg/cmd/pr/shared/commentable.go

```diff
@@ -17,7 +17,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var errNoUserComments = errors.New("no comments found for current user")
+var errNoUserComments = errors.New("no comments found for the current user")
```

Comment:
Since this error was already there I'd just leave it as is, in case some user is already  `grep`-ing against the exact message.

Reply:
thanks, reverted